envi/__init__.py

```diff
@@ -1186,21 +1186,23 @@ def executeCall(self, emu, va, args=None, ra=None):
         self.setupCall(emu, args=args, ra=ra)
         emu.setProgramCounter(va)
 
-    def execCallReturn(self, emu, value, argc):
+    def execCallReturn(self, emu, value, argc, apiReturnsVoid=False):
```

Comment:
please keep the pull requests separate.  this PR has some good value, but it will be hindered until the other one gets sorted out.  if you extricate the other PR from this one, it's more likely to get merged.

Reply:
OK, it's backed out.  This PR is just the Windows API update.